lib/handle-create-event.js

```diff
@@ -11,8 +11,10 @@ const createIssue = require('./create-issue')
 
 const template = readFileSync(pathResolve(__dirname, '..', 'instructions.md'), 'utf8')
 
-function handleCreateEvent (context) {
+async function handleCreateEvent (context) {
```

Comment:
we don’t need the `async` here. It’s only needed when you use `await` in the function

Reply:
We use await on line 16 for const config

Reply:
you are right 👍 sorry I’m to tired :D